src/app/public/modules/progress-indicator/progress-indicator-title/progress-indicator-title.component.html

```diff
@@ -1,4 +1,6 @@
-<h3
-  class="sky-section-heading sky-progress-indicator-title">
+<div
+  class="sky-progress-indicator-title sky-section-heading"
+  role="heading"
+>
   <ng-content></ng-content>
-</h3>
+</div>
```

Comment:
Matt Gregg mentioned that component "headings" should not use heading HTML tags because this can throw off screen readers (it's still okay to use heading tags in general markup such as above paragraphs, etc.). For our components, we should be using `div` with a `role` of "heading".

Reply:
Hm... I'm interested in hearing more about this. In the meantime, I'll take your word and we can move forward!